### PR TITLE
[JENKINS-31847] CLI command clear-queue extracted from core to CLI

### PR DESCRIPTION
**Pull request by pjanouse**
Please review, thx Pavel

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Merge conflict with https://github.com/jenkinsci/jenkins/pull/1938 , I suppose

**Comment by pjanouse** [comment]
Conflict resolved.

File: core/src/main/java/hudson/cli/ClearQueueCommand.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+package hudson.cli;
+
+import hudson.Extension;
+import jenkins.model.Jenkins;
+import org.acegisecurity.AccessDeniedException;
+
+import java.util.logging.Logger;
+
+/**
+ * @author pjanouse
```
Comment (KostyaSha) [inline core/src/main/java/hudson/cli/ClearQueueCommand.java]: Please wrote normal description.

File: core/src/main/java/hudson/cli/ClearQueueCommand.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import jenkins.model.Jenkins;
+import org.acegisecurity.AccessDeniedException;
+
+import java.util.logging.Logger;
+
+/**
+ * Clears the build queue
+ *
+ * @author pjanouse
+ * @since TODO
```
Comment (batmat) [inline core/src/main/java/hudson/cli/ClearQueueCommand.java]: TODO ?

File: core/src/main/java/hudson/cli/ClearQueueCommand.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        return Messages.ClearQueueCommand_ShortDescription();
+    }
+
+    @Override
+    protected int run() throws Exception {
+
+        boolean errorOccurred = false;
+
+        try {
+            Jenkins.getInstance().getQueue().clear();
```
Comment (batmat) [inline core/src/main/java/hudson/cli/ClearQueueCommand.java]: Jenkins.getActiveInstance() instead (nullability check)

File: core/src/main/java/hudson/cli/ClearQueueCommand.java (hunk 2 above)
Comment (oleg-nenashev) [inline core/src/main/java/hudson/cli/ClearQueueCommand.java]: It's a right approach for new classes. Should be fixed by a person who merges the PR

Comment (batmat) [inline core/src/main/java/hudson/cli/ClearQueueCommand.java]: Right, get it.

File: core/src/main/java/hudson/cli/ClearQueueCommand.java (hunk 3 above)
Comment (pjanouse) [inline core/src/main/java/hudson/cli/ClearQueueCommand.java]: Agree, this should be probably fixed/improved in other CLI commands as well.

**Comment by pjanouse** [comment]
Fixed Jenkins.getInstance() -> Jenkins.getActiveInstance()

**Comment by oleg-nenashev** [comment]
:+1:

**Comment by daniel-beck** [comment]
:+1:

**Comment by olivergondza** [comment]
To be amended once JENKINS-32273 is fixed.

**Comment by pjanouse** [comment]
Fixed and unified with current CLI module impl.

**Pull request merged**